src/app/about/page.jsx

```diff
@@ -1,11 +1,36 @@
 import Link from "next/link";
 import React from "react";
+import { fetchGraphQL } from '@/lib/graphql-utils';
 
-const About = () => {
+async function getAbout() {
+  const query = `
+  query About {
+    abouts {
+      content {
+        text
+      }
+      title
+      id
+    }
+  }
+  
+  `;
+
+  const data = await fetchGraphQL(query);
+  return data;
+}
+
+const About = async () => {
+  const { abouts } = await getAbout();
```

Comment:
why do you fetch many abouts? Isn't there supposed to be a single About?

Reply:
fixed